src/content/docs/style-guide/writing-docs/writer-workflow/tech-writer-workflow.mdx

```diff
@@ -55,72 +55,81 @@ Need to edit a doc? Use this table to decide where to work!
-3. Click **Commit to [yourbranchname]**
+4. Once built, preview the site in your browser by navigating to [http://localhost:8000/](http://localhost:8000/)
 
 ## 5. Publish your commits [#publish-commits]
-
-Once you have committed your changes, you're almost ready to open your Pull Request.
-
-First, you need to ensure your branch is pushed upstream.
+Once you're satisfied with doc, you'll want to push all of your commits upstream. This makes them available to everyone else in the GitHub repository. 
 
 1. On GitHub desktop, click the blue **Publish Branch** button if available.
 2. If you don't see the Publish Branch, click the blue **Push Origin** button. 
 
-This will push all your commits upstream and make them available to everyone else through the GitHub repository.
+Now you're ready for a Pull Request. 
 
 ## 6. Open your pull request [#open-pr]
-Now that your commits are available to everyone, you need to notify people that your changes are ready to be merged into the `develop` branch.
+Your commits are available to everyone, but you'll need to notify people that your changes are ready to be merged into the `develop` branch.
 
-To do this, you open a pull request:
+To do this, you open a pull request (PR):
 
 1. On GitHub Desktop, click the blue **Create Pull Request button**.
-2. This will open GitHub in your browser, and prompt you to fill in your pull request. Ensure you're merging from your branch into either the `main` or `develop` branch. If you scroll down, you can review all your commits to ensure they reflect all your changes.
+2. This will open GitHub in your browser, and prompt you to fill in your pull request. 
+    - Ensure you're merging from your branch into either the `main` or `develop` branch.
+    - Scroll down to review all your commits. They should reflect all your changes.
 3. Just like your commit description, your pull request description should be detailed and give the full context of your changes. 
     - Feel free to add any additional context here (GitHub issue link or Jira issue ID, SMEs, etc.)
-    - To request a review from another Tech Writer: in GitHub open the PR, navigate to the **Conversation** section, and then select or type in a reviewer name in the **Reviewer** section. 
-4. Add any [relevant labels](/docs/style-guide/writer-workflow/github-intro) to your PR. If you do not add `from_tw`, the PR will not be automatically assigned to another writer for review.
-5. Once you're satisfied with your pull request, click the green **Create pull request button**. 
+    - Request a review from another tech writer by opening the PR, navigating to the **Conversation** section, and inputting the reviewer's name in the **Reviewer** section. 
+4. Add any [relevant labels](/docs/style-guide/writer-workflow/github-intro) to your PR. 
+    - If you do not add `from_tw`, the PR will not be automatically assigned to another writer for review.
```

Comment:
I'm pretty sure it also needs the "content" label, is that right @bradleycamacho?

Reply:
Yeah, all PRs that touch content need a `content` label. Although, I think that is metnioned in the doc it links to.